generate-openapi-swagger.md

```diff
@@ -0,0 +1,8 @@
+# How to Generate an OpenAPI Spec for TF Operator
+
+From the root of the repo,
+`go run hack/gen-openapi-spec/main.go v0.6 > pkg/apis/tensorflow/v1beta2/swagger.json`
```

Comment:
we want to use latest v1 version  https://github.com/kubeflow/tf-operator/tree/master/pkg/apis/tensorflow/v1

Reply:
👍